<ui version="4.0" >
 <class>ManageUserDirectories</class>
 <widget class="QDialog" name="ManageUserDirectories" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Manage User Directories</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QGroupBox" name="gbDataSearch" >
     <property name="title" >
      <string>Data Search Directories</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3" >
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" >
        <item>
         <widget class="QListWidget" name="lwDataSearchDirs" />
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2" >
          <item>
           <widget class="QPushButton" name="pbAddDir" >
            <property name="text" >
             <string>Add Directory</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbRemDir" >
            <property name="text" >
             <string>Remove Directory</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbMoveUp" >
            <property name="text" >
             <string>Move Up</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbMoveDown" >
            <property name="text" >
             <string>Move Down</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="ckSearchArchive" >
        <property name="text" >
         <string>Search Data Archive</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbDefaultSave" >
     <property name="title" >
      <string>Default Save Directory</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <item>
       <widget class="QLineEdit" name="leDefaultSave" />
      </item>
      <item>
       <widget class="QPushButton" name="pbSaveBrowse" >
        <property name="text" >
         <string>Browse to Directory</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layButtons" >
     <item>
      <widget class="QPushButton" name="pbHelp" >
       <property name="maximumSize" >
        <size>
         <width>35</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>?</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbManageOtherProperties" >
       <property name="text" >
        <string>Other Properties</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbConfirm" >
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
